### PR TITLE
[ci] py-tests: run aggregate tests in CI & python tests in nightly

### PR DESCRIPTION
**Pull request by abhizer**
The aggregate tests are run in every CI invocation as they are one single program.
However, the python tests are multiple different programs and take longer, therefore are only run in the nightly CI.

Fixes: #2399

### CODE REVIEW

File: Earthfile [hunk 1]
```diff
@@ -567,7 +574,7 @@ ci-tests:
     # BUILD +test-s3
 
 nightly-tests:
-    BUILD +test-python
+    BUILD +test-python --all=1
```
Comment (ryzhyk) [inline Earthfile:577]: Shouldn't we add `BUILD +test-python` to `ci-tests` too?

**Review by ryzhyk — CHANGES_REQUESTED** [review]
How long do aggregate tests take nowadays?

**Comment by abhizer** [comment]
> How long do aggregate tests take nowadays?

Takes a min to compile and run for me.

File: Earthfile (hunk 1 above)
Comment (abhizer) [reply Earthfile:577]: I thought we already ran it as part of CI tests.

I forgot that we disabled it.

**Review by ryzhyk — APPROVED** [review]

**Pull request merged**